pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -543,13 +543,13 @@ func handleLastRebootReason(ctx *nodeagentContext) {
 	}
 
 	// if reboot stack size crosses max size, truncate
-	if len(ctx.rebootStack) > maxRebootStackSize {
+	headerSize := 100
```

Comment:
Where does the 100 come from? Better to declare it as some constant??

Reply:
declared a constant